examples/jax/smac/feedforward/decentralised/run_ippo_sep_networks.py

```diff
@@ -44,9 +44,6 @@
 def main(_: Any) -> None:
```

Comment:
Please delete this file as it is no longer valid since we removed the sep networks. Also, please remove the incorrect docstring `single_network` (3 times) in this file: https://github.com/instadeepai/Mava/blob/develop/mava/systems/jax/ippo/networks.py#L201

Reply:
Done. Thank you for catching that :call_me_hand: